.vscode/settings.json

```diff
@@ -119,5 +125,7 @@
     
     ],
     "cmake.buildDirectory": "${workspaceFolder}/build-${buildKitVendor}${buildKitVersionMajor}-${variant:cpp}-${variant:toolboxes}-${variant:mor}-${variant:buildType}-${variant:webserver}-${variant:doxygen}-${variant:matplot}",
-    "cmake.parallelJobs": 5
-}
\ No newline at end of file
+    "cmake.parallelJobs": 5,
+    "autoDocstring.docstringFormat": "numpy"
```

Comment:
why numpy?

Reply:
I looked at the list available for default docstring format. When the default one is use, the overview of it in vscode is given in the first figure, with numpy, the format is slightly different and the overview is given in the second figure.
I find this is better the second way, but of course we can choose something else !


![image](https://user-images.githubusercontent.com/60920471/222402114-16128da4-eb75-4d6c-b509-0dcb90138319.png) 
![image](https://user-images.githubusercontent.com/60920471/222402588-dacaea82-7f13-4a13-9937-a356dad58b94.png)